examples/webgpu_shadowmap_csm.html

```diff
@@ -0,0 +1,319 @@
+		</div>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.webgpu.js",
+					"three/tsl": "../build/three.webgpu.js",
+					"three/addons/": "./jsm/"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+			import { CSMShadowNode } from 'three/addons/csm/CSMShadowNode.js';
+			import { CSMHelper } from 'three/addons/csm/CSMHelper.js';
+
+			let renderer, scene, camera, orthoCamera, controls, csm, csmHelper, csmDirectionalLight;
+
+			const params = {
+				orthographic: false,
+				fade: false,
+				shadows: true,
+				far: 1000,
+				mode: 'practical',
+				lightX: - 1,
+				lightY: - 1,
+				lightZ: - 1,
+				margin: 100,
+				lightFar: 5000,
+				lightNear: 1,
+				autoUpdateHelper: true,
+				updateHelper: function () {
+
+					csmHelper.update();
+
+				}
+			};
+
+			init();
+
+			function updateOrthoCamera() {
+
+				const size = controls.target.distanceTo( camera.position );
+				const aspect = camera.aspect;
+
+				orthoCamera.left = size * aspect / - 2;
+				orthoCamera.right = size * aspect / 2;
+
+				orthoCamera.top = size / 2;
+				orthoCamera.bottom = size / - 2;
+				orthoCamera.position.copy( camera.position );
+				orthoCamera.rotation.copy( camera.rotation );
+				orthoCamera.updateProjectionMatrix();
+
+			}
+
+			function init() {
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( '#454e61' );
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 0.1, 5000 );
+				orthoCamera = new THREE.OrthographicCamera();
+
+				renderer = new THREE.WebGPURenderer( { antialias: true } );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.setAnimationLoop( animate );
+				document.body.appendChild( renderer.domElement );
+				renderer.shadowMap.enabled = params.shadows;
+				renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+
+				controls = new OrbitControls( camera, renderer.domElement );
+				controls.maxPolarAngle = Math.PI / 2;
+				camera.position.set( 60, 60, 0 );
+				controls.target = new THREE.Vector3( - 100, 10, 0 );
+				controls.update();
+
+				const ambientLight = new THREE.AmbientLight( 0xffffff, 1.5 );
+				scene.add( ambientLight );
+
+				const additionalDirectionalLight = new THREE.DirectionalLight( 0x000020, 1.5 );
+				additionalDirectionalLight.position.set( params.lightX, params.lightY, params.lightZ ).normalize().multiplyScalar( - 200 );
+				scene.add( additionalDirectionalLight );
+
+				csmDirectionalLight = new THREE.DirectionalLight( 0xffffff, 3.0 );
+
+				csmDirectionalLight.castShadow = true;
+				csmDirectionalLight.shadow.mapSize.width = 1024;
+				csmDirectionalLight.shadow.mapSize.height = 1024;
+				csmDirectionalLight.shadow.camera.near = 1;
+				csmDirectionalLight.shadow.camera.far = 20000;
```

Comment:
Why did you pick this extreme `far` value? I would expect using `1000` (`params.far`).